### DISCUSSION TITLE
Ignore `zig-cache` and `zig-out`

### DISCUSSION THREAD

**Discussion by fitrh** [nvim-telescope/telescope.nvim#1529] [Q&A] [answered]
How to ignore the directory and all its subfolders and files ?
for `node_modules`, I can add `node_modules/` inside `file_ignore_patterns`, but
it seems that it does not seems to work with `zig-cache` and `zig-out`, I have 
also tried `^zig-cache` and `zig-cache*`.

**Comment by Conni2461** [comment, marked as answer]
You should read `:help telescope.defaults.file_ignore_patterns`.

You have to escape `-`, so try `zig%-cache`

**Reply by fitrh** [reply]
Thank you, I have read them but I missed about about magic character
